### PR TITLE
Fixed typos and errors in developer community documentation

### PR DESCRIPTION
**Pull request by michizhou**
Corrected some errors in spelling and word usage in documentation for developer community guide in order to improve overall readability for use by potential developers and community members.

Signed-off-by: michizhou <myz227@nyu.edu>

### CODE REVIEW

File: developer-community/tech-talks.md [hunk 1]
```diff
@@ -38,13 +38,13 @@ Email copy:
...
+To avoid any last minute schedule changes due to potential cancellations, we should always ask someone from our team to have a long talk prepared. This way, if someone doesn't make it, there's an easy replacement available. If all else fails, please make sure to email all attendees as soon as you know this (please remember to organize a **visa** - if applicable - for this person beforehand to make sure he/she can travel if needed).
 
 ## Information & Logistics
 
 From each speaker, we'll need to collect a __Bio, Picture, Title & Abstract__ in order to build the website. 
 
-If the speaker is not based where the talk takes place, we'll need to start organising his/her trip asap so that we avoid expensive last minute tickets & hotels.
+If the speaker is not based at where the talk takes place, we'll need to start organizing his/her trip asap so that we avoid expensive last minute tickets & hotels.
```
Comment (campoy) [inline developer-community/tech-talks.md]: This `at` doesn't really make sense here, I'd say.

**Comment by campoy** [comment]
Thanks for the contribution, @michizhou!

Please make this quick change and I'll merge your PR today.

**Comment by michizhou** [comment]
@campoy I just made the change as per your request and pushed it. Please merge it today. Thanks!

**Comment by michizhou** [comment]
@campoy Just wanted to follow up with you to let you know I've already made the change suggested by your review of my PR. Please merge this PR today if you can. Thank you.

**Pull request merged**